qa/suites/rados/basic/tasks/rados_api_tests.yaml

```diff
@@ -12,6 +12,10 @@ overrides:
     - \(PG_AVAILABILITY\)
     - \(PG_DEGRADED\)
     - CEPHADM_STRAY_DAEMON
+    - do not have an application enabled
+    - POOL_APP_NOT_ENABLED
```

Comment:
The parenthesized version `\(POOL_APP_NOT_ENABLED\)` can be removed now (here and elsewhere in the PR).